src/features/auth/WalletAuthModal.tsx

```diff
@@ -121,19 +124,29 @@ export const WalletAuthModal = ({ open }: { open: boolean }) => {
   };
 
   useEffect(() => {
-    if (
-      // safe to refer to window here because we are in useEffect -g
-      window.location.search === AUTO_OPEN_WALLET_DIALOG_PARAMS ||
-      walletAuth.connectorName
-    ) {
-      walletAuth.connectorName && activate(walletAuth.connectorName);
-    }
+    if (walletAuth.connectorName && !web3Context.active)
```

Comment:
Do you understand the case when this would be true `if (walletAuth.connectorName)` ?

Reply:
it's true when there's saved walletAuth data, which happens after successful login in `useFinishAuth` [here](https://github.com/coordinape/coordinape/pull/1690/files#diff-9364f778b5d58002aee011b7f30b5acf78b68de864caa758194fb49ea89b8bccR70)